tests/robottelo/test_ssh.py

```diff
@@ -90,147 +84,87 @@ def close(self):
         """A no-op stub method."""
         self.close_ += 1
 
-    def exec_command(self, cmd, *args, **kwargs):
+    def execute(self, cmd, *args, **kwargs):
         return (self.ret_code, MockStdout(cmd, self.ret_code), MockStdout('', self.ret_code))
 
 
 class TestSSH:
     """Tests for module ``robottelo.ssh``."""
 
-    @mock.patch('robottelo.config.settings')
-    def test_get_connection_key(self, settings):
-        """Test method ``get_connection`` using key file to connect to the
-        server.
-
-        Mock up ``paramiko.SSHClient`` (by overriding method
-        ``_call_paramiko_sshclient``) before calling ``get_connection``.
-        Assert that certain parameters are passed to the (mock)
-        ``paramiko.SSHClient`` object, and that certain methods on that object
-        are called.
-        """
-        ssh._call_paramiko_sshclient = MockSSHClient
-
-        key_filename = os.path.join(os.path.abspath(__name__), 'data', 'test_dsa.key')
-        settings.server.hostname = 'example.com'
-        settings.server.ssh_username = 'nobody'
-        settings.server.ssh_password = None
-        settings.server.ssh_key = key_filename
-        settings.server.ssh_key_string = None
-        settings.server.ssh_client.command_timeout = 300
-        settings.server.ssh_client.connection_timeout = 10
-        with ssh.get_connection() as connection:
-            assert connection.set_missing_host_key_policy_ == 1
-            assert connection.connect_ == 1
-            assert connection.close_ == 0
-            assert connection.hostname == 'example.com'
-            assert connection.username == 'nobody'
-            assert connection.key_filename == key_filename
-            assert connection.password is None
-            assert connection.pkey is None
-        assert connection.set_missing_host_key_policy_ == 1
-        assert connection.connect_ == 1
-        assert connection.close_ == 1
+    def test_valid_ssh_pub_keys(self):
```

Comment:
When I moved the ssh module function to helpers, I also moved the unit test to test_helpers.py, so you can drop this when you remove the function from ssh module.